custom_md_loader/classes/JsonUtilities.cls

```diff
@@ -0,0 +1,53 @@
+/* 
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license. 
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+ 
+/**
+ * Utilities class for common manipulation of json format data
+ *
+ */
+
+public with sharing class JsonUtilities {
+
+    public class JsonUtilException extends Exception {}
+    
+    public static String JSON_BAD_FORMAT = 'The provided Json String was badly formatted.';

```

Comment:
Can you put these with the other error messages in AppConstants, so that when we convert them to label files, they'll all be together?